provider/aws/formation/rack.json

```diff
@@ -445,12 +445,10 @@
       }
     },
     "SubnetPrivate0": {
-      "Condition": "Private",
```

Comment:
AWS recommends assigning at least 2 private subnets to lambdas assigned to a VPC